src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/IndexStateManagementRestTestCase.kt

```diff
@@ -48,7 +45,7 @@ import org.elasticsearch.rest.RestStatus
 import org.elasticsearch.test.ESTestCase
 import org.elasticsearch.test.rest.ESRestTestCase
 import org.junit.rules.DisableOnDebug
-import java.util.*
+import java.util.Locale
```

Comment:
Is this import being used?

Reply:
Yes we are using in `createIndex, index.toLowerCase`. I changed from being a generic import to more specific import.